main.tf

```diff
@@ -43,23 +43,44 @@ resource "aws_security_group" "default" {
   count       = var.enabled ? 1 : 0
   name        = module.label.id
   vpc_id      = var.vpc_id
-  description = "Bastion security group (only SSH inbound access is allowed)"
+  description = "Bastion host security group"
```

Comment:
Please do not change the description.  This forces replacement of the existing resource and causes all sorts of havoc in running configurations.

Reply:
@razorsedge good point.

@DeividasJackus - I discussed with the team and it would be best to make this a variable which defaults to the old description. This way it won't break by default on upgrade but people can update the description to whatever makes sense.

Reply:
Alternately, use in the aws_security_group.default:
```
  lifecycle {
    create_before_destroy = true
  }
```